feat(store-devtools): change connectOutsideZone to be 'true' by default (#4103)
ngrx/platform: projects/ngrx.io/content/guide/store-devtools/index.md

      autoPause: true, // Pauses recording actions and state changes when the extension window is not open
      trace: false, //  If set to true, will include stack trace for every dispatched action, so you can see it in trace tab jumping directly to that part of code
      traceLimit: 75, // maximum stack trace frames to be stored (in case trace option was provided as true)

>       connectInZone: false // If set to false, the connection is established outside the Angular zone for better performance

Reviewer: ```suggestion
      connectInZone: true // If set to true, the connection is established within the Angular zone
```

      autoPause: true, // Pauses recording actions and state changes when the extension window is not open
      trace: false, //  If set to true, will include stack trace for every dispatched action, so you can see it in trace tab jumping directly to that part of code
      traceLimit: 75, // maximum stack trace frames to be stored (in case trace option was provided as true)

>       connectInZone: false // If set false true, the connection is established outside the Angular zone for better performance

Reviewer: ```suggestion
      connectInZone: true // If set to true, the connection is established within the Angular zone
```
